README.md

```diff
@@ -46,30 +46,122 @@ Simply run the following:
 
 ```
 $ go install ./cmd/cloud
+alias cloud='$HOME/go/bin/cloud'
 ```
 
-### Running
 
+### Pre-requisites
+
+- Make sure you have a key in your ~/.ssh/
+    such as:
+    ```
+    id_rsa
+    id_rsa.pub
+    ```
+    If not you need to run:
+    ```
+    ssh-keygen -t rsa -C "<your-email>"
+    ```
+
+- make sure you are running terraform 11.x version
+    ```
+    terraform version
+    ```
+    If not try to use [tfswitch](https://warrensbox.github.io/terraform-switcher/) for switching between versions easily:
+    `brew install warrensbox/tap/tfswitch`
+
+
+- Make sure you have Helm version greater/equal than 2.16.4
```

Comment:
I am pretty sure we actually have issues with newer versions of helm. That was the case in the past at least. We may just want to change this to be exactly `2.16.4` for now.

Reply:
I will change it to be indifferent from 2.16.4 as I had some issues with nginx with that version, https://stackoverflow.com/questions/60836127/error-validation-failed-serviceaccounts-nginx-ingress-not-found-serviceacc